src/styles/index.scss

```diff
@@ -1,3 +1,87 @@
+$size: 100px;
+$content-day: '';
+$start-days: (
+  'mon': 0,
+  'tue': $size + 1px,
+  'wed': $size * 2 + 2px,
+  'thu': $size * 3 + 3px,
+  'fri': $size * 4 + 4px,
+  'sat': $size * 5 + 5px,
+  'sun': $size * 6 + 6px,
+);
+$month-length: (
+  '28': 28,
+  '29': 29,
+  '30': 30,
+  '31': 31,
+);
+
+@mixin flex($flex, $justify-content, $align-items) {
+  display: $flex;
```

Comment:
The parameter `$flex` in the `flex` mixin is expected to be a boolean value (true or false) to determine if the display should be `flex` or not. However, it is being used directly as a display property value, which might not be the intended use. Consider renaming the parameter to `$display` or using a boolean value to toggle `display: flex;`.

Reply:
> The parameter `$flex` in the `flex` mixin is expected to be a boolean value (true or false) to determine if the display should be `flex` or not. However, it is being used directly as a display property value, which might not be the intended use. Consider renaming the parameter to `$display` or using a boolean value to toggle `display: flex;`.

Good point, and the name is confusing as it's only about flex, not display